### ISSUE TITLE
[Bug] Whirlpool multiple loading bars for same pool denomination

### ISSUE THREAD

**Issue by RequestPrivacy** [sparrowwallet/sparrow#606]
# Description

As far as I understand it Sparrow - or better the whirlpool client - only registers one input per pool size for remixing.
In a few instances there appear two loading bars (status "registered") in the "Mixes" column of ONE pool size. It isn't reproducible and I'm not sure what triggers the behavior or if it is really the registration of multiple inputs or rather a displaying bug. 

Some observations though: 

- It seems it hasn't to do with a recent remix
- It seems to clear (i.e. one loading bar disappears) after a new block has been found and Sparrow updates to the new height.
- It could be a problem on the whirlpool side. I can't say with 100% certainty by it might be the issue also appeared on the Samourai GUI

Nevertheless I wanted to post the issue here for potential future investigations. Once I can provide more info I will follow up...

**Comment by craigraw** [comment]
Yes, I've noticed this as well. It's something to do with the Samourai Whirlpool client, but seems harmless. Sparrow cleans up any additional bars with every new block (using the block interval as a semi-regular timer).
